.drone/drone.sh

```diff
@@ -117,30 +123,28 @@ common_install() {
   git clone https://github.com/boostorg/boost-ci.git boost-ci-cloned --depth 1
   cp -prf boost-ci-cloned/ci .
   rm -rf boost-ci-cloned
-  if [ "$boost_cache_hit" != true ]; then
-    . ./ci/common_install.sh
-    if command -v rsync &>/dev/null && [ "$drone_cache_rebuild" == true ]; then
-      if command -v apt-get &>/dev/null; then
-        apt-get install -y rsync
-      fi
-      mkdir "$cache_dir"/boost
-      rsync -a "$BOOST_ROOT/" "$cache_dir/boost"
-      echo "$boost_cache_key" >"$cache_dir/boost_cache_key.txt"
-    fi
-  else
+  if [ "$boost_cache_hit" = true ]; then
     cd ..
     mkdir boost-root
     cd boost-root
-    BOOST_ROOT="$(pwd)"
-    export BOOST_ROOT
-    ls "$cache_dir/boost"
+    export BOOST_ROOT="$(pwd)"
```

Comment:
This is effectively just removing the `ls "$cache_dir/boost"` line above.

I've been splitting commands such as `export BOOST_ROOT="$(pwd)"`  into 

```
BOOST_ROOT="$(pwd)"
    export BOOST_ROOT
```

because shellcheck gives me the error [SC2155](https://www.shellcheck.net/wiki/SC2155). Not a huge problem but still annoying because it hides more serious problems.

Reply:
shellcheck. Ok switched it back. The commits should be squashed.